openfl/federated/plan/plan.py

```diff
@@ -420,8 +446,13 @@ def get_core_task_runner(self, data_loader=None,
 
         return self.runner_
 
-    def get_collaborator(self, collaborator_name, root_certificate=None, private_key=None,
-                         certificate=None, task_runner=None, client=None, shard_descriptor=None):
+    def get_collaborator(self, collaborator_name: str,
+                         root_certificate: Union[Path, str, None] = None,
```

Comment:
Seems that we use this type often enough to create a variable for it.

Reply:
I already fixed it this way: https://github.com/intel/openfl/pull/391#discussion_r835451074